frontend/src/app/patients/PatientForm.tsx

```diff
@@ -361,92 +361,62 @@ const PatientForm = (props: Props) => {
           county,
           zipCode,
         },
-      }).then(
-        (updatedPatientFromApi: any) => {
-          showNotification(
-            toast,
-            <Alert
-              type="success"
-              title={`Your profile changes have been saved`}
-            />
-          );
+      }).then((updatedPatientFromApi: any) => {
+        showNotification(
+          toast,
+          <Alert
+            type="success"
+            title={`Your profile changes have been saved`}
+          />
+        );
 
-          const residentCongregateSetting = updatedPatientFromApi.residentCongregateSetting
-            ? "YES"
-            : "NO";
-          const employedInHealthcare = updatedPatientFromApi.employedInHealthcare
-            ? "YES"
-            : "NO";
+        const residentCongregateSetting = updatedPatientFromApi.residentCongregateSetting
+          ? "YES"
+          : "NO";
+        const employedInHealthcare = updatedPatientFromApi.employedInHealthcare
+          ? "YES"
+          : "NO";
 
-          dispatch(
-            reduxSetPatient({
-              ...updatedPatientFromApi,
-              residentCongregateSetting,
-              employedInHealthcare,
-            })
-          );
-          setSubmitted(true);
-        },
-        (error: any) => {
-          appInsights.trackException(error);
-          showError(
-            toast,
-            `${PATIENT_TERM_CAP} Data Error`,
-            "Please check for missing data or typos."
-          );
-        }
-      );
+        dispatch(
+          reduxSetPatient({
+            ...updatedPatientFromApi,
+            residentCongregateSetting,
+            employedInHealthcare,
+          })
+        );
+        setSubmitted(true);
+      });
     } else if (props.patientId) {
       trackUpdatePatient({});
       updatePatient({
         variables: {
           patientId: props.patientId,
           ...variables,
         },
-      }).then(
-        () => {
-          showNotification(
-            toast,
-            <Alert
-              type="success"
-              title={`${PATIENT_TERM_CAP} Record Saved`}
-              body="Information record has been updated."
-            />
-          );
-          setSubmitted(true);
-        },
-        (error) => {
-          appInsights.trackException(error);
```

Comment:
In this case, it seems like we're losing the app insights tracking. Should we add this to the global error handling?

Reply:
I think frontend error handling could go on the pile of tech debt. One of the better ways I have seen FE error handling is creating a function or class that abstracts away what happens (toasts, appinsights reporting) and then all the decisions on how individual errors are handled belong to that class.

Reply:
Sadly the appInsights is not currently working. The setup work was never finished for it